python/taichi/misc/image.py

```diff
@@ -2,7 +2,7 @@
 import taichi as ti
 
 
-def imwrite(img, filename):
+def imcook(img):
```

Comment:
Just curious, what does `imcook` stand for? This name looks a bit puzzling to me compared to `imwrite` or `imdisplay`